Chaincase.UI/Pages/ConnectNodePage.razor

```diff
@@ -0,0 +1,145 @@
+﻿@page "/connectfullnode"
+@using Microsoft.AspNetCore.Components
+@using Chaincase.UI.Services
+@using System.Net.Sockets
+@inject UIStateService UiStateService
+@inject StackService StackService
+@inject NavigationManager NavigationManager
+@inject ConnectNodeViewModel ConnectNodeViewModel
+@inherits ReactiveComponentBase<ConnectNodeViewModel>
+
+<IonContent class="ion-padding">
+    <div style="display:flex; flex-direction:row; justify-content: space-between; padding:10px;">
+        <IonLabel style="font-weight:600;">Connect your full node</IonLabel>
+        <IonToggle @onclick="OnToggleConnectNode" color="primary" style="padding-left:15px;" @bind-Value="ConnectNodeIsToggled"/>
+    </div>
+    <div class="ion-padding">
+        <IonLabel style="color:#999999;">
+            If you enable this option, Chaincase will allow you to manually connect to your own full node. If you disable, we will find peers to connect to for you.
+        </IonLabel>
+    </div>
+    <div class="ion-padding"> 
+        <IonItem>
+            <IonInput @bind-Value="ViewModel.NodeAddress" placeholder="Paste or scan address" disabled="@(!ConnectNodeIsToggled)" />
+            @if (ViewModel.CanConnect is { })
+            {
+                <div>
+                    <IonBadge color="@(ViewModel.CanConnect ?? false ? "success" : "warning")">
+                        @(ViewModel.CanConnect ?? false ? "Connected" : "Disconnected")
```

Comment:
can connect is not really a live status. You need to check the actual `Node.State` for this